detection_rules/config.py

```diff
@@ -100,7 +101,7 @@ def get_test_names(self, formatted: bool = False) -> (List[str], List[str]):
 
         combined_tests = sorted(set(defined_tests + self.tests_by_patterns(*patterns)))
 
-        if self.unit_tests.test_only is not None:
```

Comment:
revert this and update `UnitTest.__post_init__` assertion error to

```python
        assert (self.bypass is not None and self.test_only is not None), 'Cannot set both `test_only` and `bypass` in test_config!'
```

Reply:
im not sure if this is exactly right, but I think I get the idea.

Reply:
![Screenshot 2024-05-07 at 4 33 06 PM](https://github.com/elastic/detection-rules/assets/1636709/a8ef242a-16f8-44e0-a595-cfd5e8346004)

Reply:
Oh I see. I used the slack version. I see its updated.